wiki/shared/news/2018-02-22-osu-mania-7k-world-cup-ro16-recap/deletethis.md

```diff
@@ -0,0 +1 @@
+filler, please delete
```

Comment:
what's this? :eyes:

Reply:
well I wasn't using github desktop, so I had to create a folder somehow
not sure how to make a folder otherwise.

Reply:
hmm can you delete it?